src/o365/graph/commands/schemaextension/schemaextension-get.ts

```diff
@@ -0,0 +1,116 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import * as request from 'request-promise-native';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import GraphCommand from '../../GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class GraphSchemaExtensionGet extends GraphCommand {
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_GET}`;
+  }
+
+  public get description(): string {
+    return 'Gets information about a Microsoft Graph schema extension';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    auth
+      .ensureAccessToken(auth.service.resource, cmd, this.debug)
+      .then((): request.RequestPromise => {
+        if (this.verbose) {
+          cmd.log(`Gets schema extension with id '${args.options.id}'...`);
+        }
+
+        const requestOptions: any = {
+          url: `${auth.service.resource}/v1.0/schemaExtensions/${args.options.id}`,
```

Comment:
Could you please escape the user input for `id` in the url?

`${auth.service.resource}/v1.0/schemaExtensions/${args.options.id}` can be changed to
`${auth.service.resource}/v1.0/schemaExtensions/${encodeURIComponent(args.options.id)}`